cartography/models/aws/ec2/networkinterfaces.py

```diff
@@ -1,5 +1,9 @@
 from dataclasses import dataclass
 
+from cartography.models.aws.ec2.networkinterface_instance import EC2NetworkInterfaceToAWSAccount
+from cartography.models.aws.ec2.networkinterface_instance import EC2NetworkInterfaceToEC2Instance
+from cartography.models.aws.ec2.networkinterface_instance import EC2NetworkInterfaceToEC2SecurityGroup
+from cartography.models.aws.ec2.networkinterface_instance import EC2NetworkInterfaceToEC2Subnet
```

Comment:
Do you think that I should redefine these rel classes rather than import them?

Reply:
I think we can leave it for now, but if the API responses change and refactoring becomes problematic, we should then redefine

Reply:
Yeah, makes sense. I think it's ok to continue to import the relationship schemas since I don't think it's super likely for the ID fields that they rely on to change. We'll revisit if this ends up being a problem.